core/src/main/java/com/graphhopper/routing/util/spatialrules/SpatialRule.java

```diff
@@ -0,0 +1,76 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.util.spatialrules;
+
+import java.util.List;
+
+/**
+ * Defines rules that are valid for a certain region, e.g. a country.
+ * A rule might makes access, max-speed, etc. dependent on this region.
+ *
+ * @author Robin Boldt
+ */
+public interface SpatialRule {
+
+    double getMaxSpeed(String highway, double _default);
+
+    AccessValue getAccessible(String highwayTag, String transportationMode, AccessValue _default);
+
+    List<Polygon> getBorders();
+
+    SpatialRule setBorders(List<Polygon> borders);
+
+    SpatialRule addBorder(Polygon polygon);
+
+    /**
+     * Get the unique name for this rule, e.g. the ISO name of the country.
+     */
+    String getUniqueName();
```

Comment:
Yes, at first I did not want to call it id, but this should be easier to understand and also imply that it should be unique. But I think it makes sense to rename it.